runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -299,8 +318,10 @@ private void killedOne()
 			return;
 		}
 
-		// update counter
+		// add and update counter, set timer
+		addCounter();
 		counter.setText(String.valueOf(amount));
+		setInfoTimer(Instant.now());
```

Comment:
I would probably just move this to `addCounter`, because you really only need the timer if the counter is actually added (it has some config/null checks in the method).

Reply:
That was how it was originally and Adam told me to move it out and into the two seperate methods

Reply:
Oh I see, my bad on that. Yeah it won't get called unless you add it in that method before the `null` check on `counter`. Probably just ignore what I said and keep what adam said :+1: